infrastructure/oss/issue.go

```diff
@@ -60,6 +60,9 @@ func toIssue(
 					affectedFilePath,
 					issueRange,
 				},
+				GroupingKey:   codeAction.GroupingKey,
```

Comment:
hmm that's why...
Maybe we can then have "AdditionalProperties" or smth similar in CommandData for these cases?